public/lesswrong-editor/src/editorConfigs.js

```diff
@@ -87,6 +87,17 @@ const embedConfig = {
 				`
 			}
 		},
+		{
+			name: 'Estimaker',
+			url: /^estimaker\.app\/_\/([\w-]+).*/,
+			html: ([match, slug]) => {
+				return `
+					<div data-owid-slug="${slug}" class="estimaker-preview">
```

Comment:
nit:
```suggestion
					<div data-estimaker-slug="${slug}" class="estimaker-preview">
```
I suppose this should be disallowed by the check in packages/lesswrong/lib/vulcan-lib/utils.ts because of this... but apparently it isn't

Reply:
Removed! Turns out these data attributes weren't necessary